### DISCUSSION TITLE
Online Course on Tabular Data Protection in Business Statistics

### DISCUSSION THREAD

**Discussion by surfing-in-the-sun** [sdcTools/UserSupport#215] [User Group]
Applications for the online course on **Tabular Data Protection in Business Statistics** are now open.

The course will be held on **October 6-8, 2021 online** and a detailed description can be found in this [**pdf**](https://sdctools.github.io/UserSupport/TabularDataProtectionBusinessStatistics.pdf). 

This course shares content with the ESTP course on Statistical Disclosure Control, which was held in early 2021, but will focus deeper on the specifics of business statistics. As was also the case for this year’s ESTP course on SDC, there is a limited number of spots available. Applicants that were not accepted for the ESTP course on SDC, will be given priority in this course.

Please apply for the course [**here**](https://sdctools.github.io/UserSupport/ContactFormBusinessCourse.html) until September 24, 2021 if you are interested.

**Comment by r-ballard** [comment]
Is there a cost associated with this training? What timezone will the training be conducted in? Thank you!

**Comment by ppdewolf** [comment]
@r-ballard There are no costs associated to this course. It is funded by a European project STACE, WP2 Center of Excellence on SDC.
It will be a fully on-line course. The time zone is Central European Time (Amsterdam). Exact times still need to be fixed, but each day will likely start around 9:00 or 9:30.
